systemtests/bin/upload-symbols.py

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env python
```

Comment:
I decided to do this and `upload_symbols_by_download.py` as separate scripts even though they test the same API endpoint. It was either that or have subcommands and two modes. I started with that and then decided I wasn't saving much by combining the two.